skein/core.py

```diff
@@ -687,13 +687,15 @@ def from_current(cls):
         """
         address = _get_env_var(ADDRESS_ENV_VAR)
 
-        local_dirs = _get_env_var('LOCAL_DIRS')
         container_id = _get_env_var('CONTAINER_ID')
-        container_dir = os.path.join(local_dirs, container_id)
-        security = Security(os.path.join(container_dir, ".skein.crt"),
-                            os.path.join(container_dir, ".skein.pem"))
-
-        return cls(address, security=security)
+        for local_dir in _get_env_var('LOCAL_DIRS').split(','):
+            container_dir = os.path.join(local_dir, container_id)
+            crt_path = os.path.join(container_dir, '.skein.crt')
+            pem_path = os.path.join(container_dir, '.skein.pem')
+            if os.path.exists(crt_path) and os.path.exists(pem_path):
+                return cls(address, security=Security(crt_path, pem_path))
+
+        raise RuntimeError("Failed to resolve .skein.{crt,pem} in 'LOCAL_DIRS'")
```

Comment:
Can you raise `FileNotFoundError` instead?

Reply:
I think this is Python 3.X only, Python 2.X uses `IOError` in the same context.

Reply:
Correct, there's a compatibility shim on top for skein (the error is already imported at the top of this file, just need to use it here).